pep-9999.txt

```diff
@@ -0,0 +1,163 @@
+PEP: 9999
+Title: Metadata for Python Software Packages 1.3
+Version: $Revision$
+Last-Modified: $Date$
+Author: Dustin Ingram <di@di.codes>
+Discussions-To: distutils-sig <distutils-sig at python.org>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 1-Dec-2017
+Python-Version: 3.x
+Post-History:
+Replaces: 345
+
+
+Abstract
+========
+
+This PEP describes a mechanism for adding metadata to Python distributions.
+It includes specifics of the field names, and their semantics and
+usage.
```

Comment:
This opening paragraph should change. Something like:

> This PEP describes the changes between versions 1.2 and 1.3 of the core metadata specification for Python packages. It also changes to the canonical source for field specifications to the `Core Metadata Specification`_ reference page on packaging.python.org.

And then adjust the rest of the abstract accordingly (e.g. while mentioning PEP 345 is good, I don't think you need to mention the others)